src/components/home/LearnStack.js

```diff
@@ -0,0 +1,74 @@
+import React, { useRef }from 'react'
+import { Animated, Dimensions, StyleSheet, TouchableOpacity, Text, View } from 'react-native'
+import { LinearGradient } from 'expo-linear-gradient'
+
+import buttonSound from 'assets/sounds/button2.mp3'
+import useMusicContext from 'contexts/Music'
+
+const { width } = Dimensions.get('window')
+const BUTTON_SIZE = width * .7
+const FONT_SIZE = BUTTON_SIZE * .1
+
+const LearnStack = ({ navigation }) => {
+	const playButtonAnim = useRef(new Animated.Value(1)).current
+  const { playAudioAsync } = useMusicContext()
+
+  const onPressEvent = async() => {
+    await playAudioAsync(buttonSound)
+    navigation.navigate('LearnStack', { screen: 'learn' })
+  }
+
+	return (
+    <TouchableOpacity onPressIn={onPressEvent} style={{transform: [{ scale: playButtonAnim }]}} activeOpacity={.6}>
+      <LinearGradient
+        colors={['blue', 'purple']}
+        style={[styles.buttonContainer, {width: BUTTON_SIZE}]}
+      >
+        <View style={styles.innerButtonContainer}>
+          <View style={styles.relief} />
+          <Text style={styles.buttonText}>Learn</Text>
```

Comment:
- add translations for any text you add, see src/locales
- naming of this component as mentioned above

Reply:
T'es pas ma mère 🙄 



Mais c'est fait...